File: src/Z.Core/System.Double/System.Math/Double.Sinh.cs
// Description: C# Extension Methods | Enhance the .NET Framework and .NET Core with over 1000 extension methods.
// Website & Documentation: https://csharp-extension.com/
// Issues: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License (MIT): https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: https://zzzprojects.com/
// Copyright © ZZZ Projects Inc. All rights reserved.
using System;

public static partial class Extensions
{
    /// <summary>
    ///     Returns the hyperbolic sine of the specified angle.
    /// </summary>
    /// <param name="value">An angle, measured in radians.</param>
    /// <returns>The hyperbolic sine of . If  is equal to , , or , this method returns a  equal to .</returns>
    public static Double Sinh(this Double value)
    {
        return Math.Sinh(value);
    }
}

File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Core/System.Double/System.Math/Double.Sinh.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library


Public Module Extensions_224

	''' <summary>
	'''     Returns the hyperbolic sine of the specified angle.
	''' </summary>
	''' <param name="value">An angle, measured in radians.</param>
	''' <returns>The hyperbolic sine of . If  is equal to , , or , this method returns a  equal to .</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function Sinh(value As [Double]) As [Double]
		Return Math.Sinh(value)
	End Function
End Module


